envs/dali-env.yaml

```diff
@@ -4,18 +4,21 @@ imported_envs:
 {% endif %}
 
 packages:
-{% if build_type == 'cuda' %}
-  - feedstock : cudatoolkit    #[cudatoolkit == "11.2"]
-  - feedstock : opencv
-  - feedstock : https://github.com/conda-forge/clangdev-feedstock     # [ppc64le]
-    git_tag: 98e30dba8b405a388d5b5d0fcca3834ee40b0b1c                 # [ppc64le]
-  - feedstock : DALI
-  - feedstock : jpeg-turbo
-  - feedstock : libflac
-  - feedstock : libsndfile
-  - feedstock : https://github.com/conda-forge/libtar-feedstock
-    patches :
-      - feedstock-patches/libtar/0001-Changed-libtar-source-url-as-older-one-is-not-availa.patch
+{% if not s390x %}
+  {% if build_type == 'cuda' %}
```

Comment:
As build_type can only be cpu and not cuda for s390x, line 7 may be redundant. Would not harm though.